public/app/routes/routes.tsx

```diff
@@ -374,75 +374,216 @@ export function getAppRoutes(): RouteDescriptor[] {
       path: '/integrated-alerting',
+      // eslint-disable-next-line react/display-name
+      component: () => <Redirect to="/integrated-alerting/alerts" />,
+    },
+    {
+      path: '/integrated-alerting/alerts',
+      component: SafeDynamicImport(
+        () =>
+          import(
+            /* webpackChunkName: "IntegratedAlertingAlerts" */ 'app/percona/integrated-alerting/components/Alerts/Alerts'
+          )
+      ),
+    },
+    {
+      path: '/integrated-alerting/alert-rule-templates',
+      component: SafeDynamicImport(
+        () =>
+          import(
+            /* webpackChunkName: "IntegratedAlertingTemplates" */ 'app/percona/integrated-alerting/components/AlertRuleTemplate/AlertRuleTemplate'
+          )
+      ),
+    },
+    {
+      path: '/integrated-alerting/notification-channels',
       component: SafeDynamicImport(
         () =>
           import(
-            /* webpackChunkName: "IntegratedAlertingPage" */ 'app/percona/integrated-alerting/IntegratedAlertingPage'
+            /* webpackChunkName: "IntegratedAlertingChannels" */ 'app/percona/integrated-alerting/components/NotificationChannel/NotificationChannel'
           )
       ),
     },
     {
-      path: '/integrated-alerting/:tab',
+      path: '/integrated-alerting/alert-rules',
       component: SafeDynamicImport(
         () =>
           import(
-            /* webpackChunkName: "IntegratedAlertingPage" */ 'app/percona/integrated-alerting/IntegratedAlertingPage'
+            /* webpackChunkName: "IntegratedAlertingRules" */ 'app/percona/integrated-alerting/components/AlertRules/AlertRules'
           )
       ),
     },
     {
       path: '/dbaas',
-      component: SafeDynamicImport(() => import(/* webpackChunkName: "DbaasPage" */ 'app/percona/dbaas/DBaaS')),
+      // eslint-disable-next-line react/display-name
+      component: () => <Redirect to="/dbaas/kubernetes" />,
+    },
+    {
+      path: '/dbaas/kubernetes',
+      component: SafeDynamicImport(
+        () =>
+          import(
+            /* webpackChunkName: "DbaaSKubernetesPage" */ 'app/percona/dbaas/components/Kubernetes/KubernetesInventory'
+          )
+      ),
     },
     {
-      path: '/dbaas/:tab',
-      component: SafeDynamicImport(() => import(/* webpackChunkName: "DbaasPage" */ 'app/percona/dbaas/DBaaS')),
+      path: '/dbaas/dbclusters',
+      component: SafeDynamicImport(
+        () => import(/* webpackChunkName: "DbaaSClustersPage" */ 'app/percona/dbaas/components/DBCluster/DBCluster')
+      ),
     },
     {
       path: '/backup',
-      component: SafeDynamicImport(() => import(/* webpackChunkName: "BackupPage" */ 'app/percona/backup/BackupPage')),
+      // eslint-disable-next-line react/display-name
+      component: () => <Redirect to="/backup/inventory" />,
     },
     {
-      path: '/backup/:tab',
-      component: SafeDynamicImport(() => import(/* webpackChunkName: "BackupPage" */ 'app/percona/backup/BackupPage')),
+      path: '/backup/inventory',
```

Comment:
It's become much more readable for me now =))))

Reply:
Yes, more explicit now )